public/static/docs/api-reference/open.md

```diff
@@ -0,0 +1,173 @@
+### `dvc.api.open(path, repo=None, rev=None, remote=None, mode="r", encoding=None)`
+
+Opens a tracked file.
+
+```py
+import dvc.api
+
+with dvc.api.open(
+        'get-started/data.xml',
+        repo='https://github.com/iterative/dataset-registry'
+        ) as fd:
+    # ... fd is a file descriptor that can be processed normally.
+```
+
+All **parameter** types are
+[string](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str).
+
+## Description
+
+Open file or model (`path`) tracked in a <abbr>DVC projects</abbr> (by DVC or
+Git), and return a corresponding
+[file object](https://docs.python.org/3/glossary.html#term-file-object). If the
+file cannot be found, an `PathMissingError` is raised.
+
+> This function is analogous to the
+> [`open()`](https://docs.python.org/3/library/functions.html#open) Python
+> builtin.
+
+It may only be used as a
+[context manager](https://www.python.org/dev/peps/pep-0343/#context-managers-in-the-standard-library)
+(using the `with` keyword, as shown in the [Examples](#examples) below).
+
+> Use `dvc.api.read()` to get the file's contents directly – no _context
+> manager_ involved.
+
+`dvc.api.open()` reads (streams) the file trough a direct connection to the
+storage whenever possible, so it does not require any space on the disc to save
+the file before making it accessible. The only exception is when using a Google
+Drive [remote type](/doc/command-reference/remote/add#supported-storage-types).
+
+## Parameters
+
+- **`path`** - location and file name of the file in `repo`, relative to the
+  project's root.
+
+- `repo` - specifies the location of the DVC project. If not supplied, defaults
+  to the current DVC project. It can be a URL or a file system path. Both HTTP
+  and SSH protocols are supported for online Git repos (e.g.
+  `[user@]server:project.git`).
+
+  A `dvc.api.UrlNotDvcRepoError` is raised if `repo` is not a valid DVC project.
+
+- `rev` - Git commit (any [revision](https://git-scm.com/docs/revisions) such as
+  a branch or tag name, or a commit hash). If not supplied, it uses the default
+  Git revision, `HEAD`. If `repo` is a Git repo, this option is ignored.
+
+- `remote` - name of the [DVC remote](/doc/command-reference/remote) to look for
+  the target data. If not supplied, the cache directory is tried first for local
+  projects; The default remote of `repo` is tried otherwise.
+
+  A `dvc.exceptions.NoRemoteError` is raised if no `remote` is found.
+
+- `mode` - specifies the mode in which the file is opened. Defaults to `"r"`
+  (read). Mirrors the namesake parameter in builtin
+  [`open()`](https://docs.python.org/3/library/functions.html#open).
+
+- `encoding` - used to decode the file contents to a string. This should only be
+  used in text mode. Defaults to `"utf-8"`. Mirrors the namesake parameter in
+  builtin `open()`.
```

Comment:
@Suor is the actual effective default `"utf-8"`? In https://docs.python.org/3/library/functions.html#open it reads

> if encoding is not specified the encoding used is platform dependent: `locale.getpreferredencoding(False)` is called to get the current locale encoding.